app/src/test/java/org/simple/clinic/removeoverdueappointment/RemoveOverdueUiRendererTest.kt

```diff
@@ -28,4 +28,22 @@ class RemoveOverdueUiRendererTest {
     verify(ui).disableDoneButton()
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `render selected remove appointment reason`() {
```

Comment:
Can we rename this test method to better reflect the use case? Ex: "when a cancel reason is selected, the 'done' button must be enabled".

This comment also applies to the test name in the previous commit.